forge-gui/res/cardsfolder/upcoming/sword_coast_serpent_capsizing_wave.txt

```diff
@@ -0,0 +1,17 @@
+Name:Sword Coast Serpent
+ManaCost:5 U U
+Types:Creature Serpent Dragon
+PT:6/6
+S:Mode$ Continuous | Affected$ Card.Self | AddHiddenKeyword$ Unblockable | CheckSVar$ X | SVarCompare$ GE1 | Description$ CARDNAME can't be blocked as long as you've cast a noncreature spell this turn.
+SVar:X:Count$ThisTurnCast_Card.nonCreature+YouOwn
```

Comment:
Needs to be YouCtrl for the rare corner case that opp managed to cast one of yours!